### PR TITLE
[MAINT-546] Changing _link to _trackEvent to avoid redirection

### PR DESCRIPTION
**Pull request by raveneyex**
Jira Tickets
-----------------
https://jira.nationalgeographic.com/browse/MAINT-543
https://jira.nationalgeographic.com/browse/MAINT-546

The Problem
------------------
Some links were tracked by analytics using the _link function that causes a redirect. This resulted in trouble for links with target="_blank"

The Solution
-----------------
Change the _link function for the _trackEvent function and add a label to differentiate this trackings.

How to Test
-----------------
Go to the main page of the app
Open the network tab
Verify that analytics.min.js is loaded
Verify that analytics is still tracking events in the page (scroll, engagement, dwell, bounce, etc).

### CODE REVIEW

**Comment by chuckcarpenter** [comment]
Why not just add nationalgeographiclodges.com to https://github.com/natgeo/modules-analytics/blob/feature/analytics-non-redirect/analytics.coffee#L117 ?

File: analytics.coffee [hunk 1]
```diff
@@ -207,7 +207,7 @@ init = (options) ->
 
         # NatGeo TLDs
         if siteName(targetA.href) in settings.natGeoDomains
-            track ['_link', targetA.href]
+            track ['_trackEvent', 'engagement', 'natgeo-domains-click', targetA.title or siteName(targetA.href) ]
```
Comment (chuckcarpenter) [inline analytics.coffee]: Better question, how is it hitting this if it's not in the natGeoDomains list?

**Comment by raveneyex** [comment]
@chuckcarpenter because the _link function will still cause a redirect in the current tab, and they are counting on having those links opening in a new tab.
In addition, the problem is not when visiting a link to nationalgeographiclodges.com; it is when visiting a link from nationalgeographiclodges.com to other natgeo domains.

**Comment by chuckcarpenter** [comment]
makes sense @raveneyex, but the tickets you link referencing the file http://images.nationalgeographic.com/wpf/sites/common/j/analytics.js and that is not this module.

**Comment by jaanauati** [comment]
So @chuckcarpenter, do you think wen can merge this one?

Thanks!

**Comment by chuckcarpenter** [comment]
@jaanauati I have no issue w/ the code, but was wondering what the response to this was (Since the reporting changes) https://jira.nationalgeographic.com/browse/MAINT-543?focusedCommentId=289793&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-289793

**Comment by jaanauati** [comment]
@chuckcarpenter understood, I asked Mia again. Thanks!

**Pull request closed without merging**